index.js

```diff
@@ -604,7 +595,7 @@ PersonaClient.prototype.updateProfile = function(guid, profile, token, callback)
                 'Authorization': 'Bearer ' + token
             },
             data:{
-               profile:JSON.stringify(profile)
+                profile:JSON.stringify(profile)
```

Comment:
Does profile need to used with `JSON.stringify` before handing it to `http.request`?